app/lib/import_export_constants.rb

```diff
@@ -59,7 +59,8 @@ module ImportExportConstants # rubocop:todo Metrics/ModuleLength
                                 lab_2_result jurisdiction_path assigned_user gender_identity sexual_orientation race_other race_unknown
                                 race_refused_to_answer vaccine_1_group_name vaccine_1_product_name vaccine_1_administration_date vaccine_1_dose_number
                                 vaccine_1_notes vaccine_2_group_name vaccine_2_product_name vaccine_2_administration_date vaccine_2_dose_number
-                                vaccine_2_notes follow_up_reason follow_up_note].freeze
+                                vaccine_2_notes vaccine_3_group_name vaccine_3_product_name vaccine_3_administration_date vaccine_3_dose_number vaccine_3_notes
```

Comment:
I'm not sure if shifting down `follow_up_reason` and `follow_up_note` is what we want. @holmesie @tstrass @hackrm thoughts?

Reply:
We do not want to shift the existing field in the import template, so the additional vaccine fields will just have to be tacked onto the end.

Reply:
I've never added anything to import before, do you mind explaining why?

Reply:
I think this is just to make it so that users don't have to shift existing columns down